docs/projects/reviews/2024/2024-vc-api-implementation-annual.md

```diff
@@ -0,0 +1,79 @@
+# 2024 VC API Implementation
+
+## Project Health
+
+The project remains healthy.
+Contributions for the initial portions of the year were focused on housekeeping items, however the project currently has development activity for functional and non-functional changes to increase the likelihood of further adoption and impact in the year ahead.
+
+Contributions have been made by two organizations, Energy Web and Impactility.
+Several forks have been made in the past year, suggesting some initial community attention.
+
+The underlying [VC API specification](https://w3c-ccg.github.io/vc-api/) (currently developed in the context of the W3C Credentials Community Group) is active and the specification group has the intention of bringing the specification to a W3C standards track in the months ahead.
+
+## Maintainer Diversity
+
+The project currently has four maintainers, from two organizations, Energy Web & Impactility.
+
+## Project Adoption
+
+The project currently one primary adoption organization, Elia Group. Elia continues to showcase the VC API project on their SSI demonstration site (https://ssi4energy.eliagroup.eu/en).
```

Comment:
It looks like the verb is missing "The project currently **has** one primary adoption organization, Elia Group"

Reply:
Thanks @skounis , fixed pushed here: [8078832](https://github.com/openwallet-foundation/tac/pull/185/commits/8078832996a417219641d4ef8f331ef354b2c9b4)